pyproject.toml

```diff
@@ -26,8 +26,11 @@ classifiers = [
     "Programming Language :: Python :: 3",
 ]
 dependencies = [
+    "escapism>=1.0",
     "jupyterhub>=1.2",
+    "oauthenticator>=0.13.0",
     "oauthlib>=3.1",
+    "PyJWT[crypto]>=2.4.0",
```

Comment:
- I tried using pyjwt 2.4.0 and oauthenticator 0.13.0 etc in the test suite and found pyjwt 2.6.0 was required for all tests to pass (one test failure was a validation exception that was expected, but not raised).
- I see oauthenticator 15.1.0 still requires jupyterhub 1.2 only, so we can introduce it without forcing up the jupyterhub version
- I bumped escapism to the latest release, one patch version
- I saw a CVE patched in oauthlib 3.2.2 and it shouldn't be a breaking change to require it

```suggestion
    "escapism>=1.0.1",
    "jupyterhub>=1.2",
    "oauthenticator>=15.1.0",
    "oauthlib>=3.2.2",
    "PyJWT[crypto]>=2.6.0",
```

Like this, tests should pass even if we use the oldest version we declare compatibility with. Also, we don't make a breaking change with the oauthenticator bump as it wasn't required before, and we can feel confident we don't risk users being exposed to known vulnerabilities etc.

Reply:
Thanks for spotting this! I've overlooked to test against lowest specified dependency version. What is your workflow for doing this?